config/application.rb

```diff
@@ -41,6 +41,9 @@ class Application < Rails::Application
     # User will be timed out after 15 minutes of inactivity
     config.session_inactivity = 15.minutes
 
+    # The cache reload time for the JWKS file from amazon
+    config.jwks_expiry = 1.hour
```

Comment:
without the comment, the key name is somewhat misleading, could we call it something like `config.jwks_cache_expiry` or similar?

Reply:
Changed